projects/packages/my-jetpack/_inc/components/product-cards-section/protect-card/protect-value-section.tsx

```diff
@@ -113,10 +113,19 @@ const ValueSection: FC< {
 				<div className="value-section__logins-blocked">
 					<div className="value-section__heading">Logins Blocked</div>
 					<div className="value-section__data">
-						<div className="logins_blocked__count">0</div>
+						<LoginsBlockedStatus />
 					</div>
 				</div>
 			</div>
 		</>
 	);
 };
+
+const LoginsBlockedStatus = () => {
+	const {
+		protect: { wafConfig: wafData },
+	} = getMyJetpackWindowInitialState();
+	const { blocked_logins: blockedLoginsCount } = wafData;
```

Comment:
```suggestion
	const { blocked_logins: blockedLoginsCount = 0 } = wafData || {};
```

Might want to add that just to ensure we don't get an error here